packages/webapp/src/components/CardWithStatus/TaskCard/styles.module.scss

```diff
@@ -53,6 +53,13 @@
   border-bottom: 1px solid var(--teal700);
 }
 
+.iconTextContainerDisabled {
```

Comment:
I'm not sure if `iconTextContainerDisabled` is the right word here because the container is not disabled. Maybe we can change it to `iconTextContainerNoUnderline` or `iconTextContainerUneditable`?

Reply:
Thanks for the feedback! Changed `iconTextContainerDisabled` to `iconTextContainerNoUnderline`